nav2_map_server/include/nav2_map_server/map_server.hpp

```diff
@@ -21,19 +23,62 @@
 namespace nav2_map_server
 {
 
+/**
+ * @class nav2_map_server::MapServer
+ * @brief This class on activate, loads the OccGridLoader node to start map
+ * server from the yaml file which is being passed as parameter.
+ */
 class MapServer : public nav2_util::LifecycleNode
 {
 public:
+  /**
+   * @brief A constructor for nav2_map_server::MapServer
+   */
   MapServer();
+
+  /**
+   * @brief A Destructor for nav2_map_server::MapServer
+   */
   ~MapServer();
 
 protected:
-  // Implement the lifecycle interface
+  /**
+   * @brief Sets up required params and calls OccGridLoader node's configure
+   * state
+   * @param state Lifecycle Node's state
+   * @return Success or Failure
+   */
   nav2_util::CallbackReturn on_configure(const rclcpp_lifecycle::State & state) override;
+  /**
+   * @brief Calls OccGridLoader node's activate state
+   * @param state Lifecycle Node's state
+   * @return Success or Failure
+   */
   nav2_util::CallbackReturn on_activate(const rclcpp_lifecycle::State & state) override;
+  /**
+   * @brief Calls OccGridLoader node's deactivate state
+   * @param state Lifecycle Node's state
+   * @return Success or Failure
+   */
   nav2_util::CallbackReturn on_deactivate(const rclcpp_lifecycle::State & state) override;
+  /**
+   * @brief Calls OccGridLoader node's cleaningup state and resets the memeber
```

Comment:
`member` variables, typo, along with `cleaningup` to cleaning up

Reply:
Cool! On it